docs/platforms/dart/integrations/http-integration.mdx

```diff
@@ -21,6 +21,23 @@ var uriResponse = await client.post('https://example.com/whatsit/create',
 }
 ```
 
+When an error occurs, the following information is captured and sent to Sentry:
+
+Request details:
+
+- Method
+- URL
+- Headers
+- Body
+- Content length
+- Duration
+
+Response details:
+
+- Headers
+- Content length
+- Status code
```

Comment:
`m`: can we indicate which of these are subject to scrubbing of sensitive data per default? and maybe add a link to https://docs.sentry.io/platforms/flutter/data-management/sensitive-data/